src/components/steps/DecryptXprv.vue

```diff
@@ -99,8 +100,10 @@ export default {
         backupPassword: this.password,
       })
       if (!this.xprvError) {
-        await this.setXprv({ result: this.fileInfo.data.master_key })
-        await this.setBackupPassword({ result: this.password })
```

Comment:
Removed this await intentionally?

Reply:
Here we are calling to Vuex mutations which are synchronous so await is unnecessary